pages.nl/android/am.md

```diff
@@ -0,0 +1,20 @@
+# am
+
+> Android-activiteitenmanager.
+> Meer informatie: <https://developer.android.com/studio/command-line/adb#am>.
+
+- Start een specifieke activiteit:
+
+`am start -n {{com.android.settings/.Settings}}`
+
+- Start een activiteit en geef er gegevens aan door:
+
+`am start -a {{android.intent.action.VIEW}} -d {{tel:123}}`
+
+- Start een activiteit die overeenkomt met een specifieke actie en [c]ategorie:
```

Comment:
```suggestion
- Start een activiteit die overeenkomt met een specifieke actie en categorie:
```
I would suggest removing `[]` as the native equivalent word isn't a supported long option.